flask_jsonapi/resource_repository_views.py

```diff
@@ -26,7 +26,7 @@ def update(self, id, data, **kwargs):
 
 
 class ResourceRepositoryListView(ResourceRepositoryViewMixin, resources.ResourceList):
-    def read_many(self, filters):
+    def read_many(self, filters, pagination):
         return self.repository.get_list(filters)
 
```

Comment:
def get_count(self, filters):
        return self.repository.get_count(filters)

Reply:
added in last commit